05/app.js

```diff
@@ -0,0 +1,50 @@
+function Student(name, surname) {
+	this.firstName = name;
+	this.lastName = surname;
+	this.grades = {};
+}
+
+Student.prototype.addGrade = function (subject, grade) {
+	if (typeof this.grades[subject] === 'undefined') {
+		this.grades[subject] = [];
```

Comment:
Formatowanie :) 
Z tego co zauważyłem to chyba korzystać z jakiegoś gotowego "beautifaja" - musisz go lepiej skonfigurować, a najlepiej, aby nie modyfikował kodu napisanego przez kogoś innego tj. nie przez Ciebie bo inaczej zmienia się wszystko i nie widać w kodzie Twoich faktycznych zmian, a wszystko jest oznaczane jako zmodyfikowane.